great_expectations/rule_based_profiler/types/data_assistant_result/onboarding_data_assistant_result.py

```diff
@@ -4,4 +4,8 @@
 
 
 class OnboardingDataAssistantResult(DataAssistantResult):
-    pass
+    # A mapping is defined for which metrics to plot and their associated expectations
+    EXPECTATION_METRIC_MAP = {
+        "expect_table_row_count_to_be_between": "table.row_count",
+        "expect_column_unique_value_count_to_be_between": "column.distinct_values.count",
+    }
```

Comment:
Same note here - if it unavoidable then no worries.

Reply:
@cdkini I think it is unavoidable, but we should add the type hint to the constant -- @NathanFarmer -- thanks!

Reply:
@alexsherstinsky adding a type hint will throw `Mutable default ... is not allowed` error due to this being a dataclass. I suspect this is why you did not type hint the `ALLOWED_KEYS` set directly below this constant.